internal/configs/annotations.go

```diff
@@ -98,6 +98,14 @@ var validPathRegex = map[string]bool{
 	"exact":            true,
 }
 
+// List of Ingress Annotation Keys used by the Ingress Controller
+var allowedAnnotationKeys = []string{
```

Comment:
nit: Can we store this as a map, with empty values so its like a set instead, so we get O(1) reads, instead of iterating through the keys

```go
allowedAnnotationKeys := map[string]struct{}{
		"nginx.org":                          {},
		"nginx.com":                          {},
		"f5.com":                             {},
		"ingress.kubernetes.io/ssl-redirect": {},
	}
```